scripts/insert_aws_org_tree.py

```diff
@@ -101,7 +108,12 @@ def _insert_account(self, org_node, act_info, date):
                 org_node["level"],
                 account_alias_id,
             ]
+            self.account_alias_mapping[account_alias_id] = act_info
             self.cursor.execute(insert_account_sql, values)
+            LOG.info(
+                f"Adding account={act_info['account_alias_name']} "
```

Comment:
How do you feel about wrapping the `account_alias_name` in quotations to make it easier to read since the examples have spaces in them? Ie. changing it from: 
```
Adding account=Root A Test to org=R_001 on created_timestamp=2020-04-07
```
to 
```
Adding account="Root A Test" to org=R_001 on created_timestamp=2020-04-07
```

Reply:
I love that idea!

Reply:
That is a great idea @abaiken I will make that change real quick and push it up.

Reply:
Example of new log output:
```
INFO:__main__:Converted date_delta=-90 to 2020-04-07
INFO:__main__:Creating OU=R_001 with path=R_001 at level=0 on created_timestamp=2020-04-07.
INFO:__main__:Adding account='Root A Test' to org=R_001 on created_timestamp=2020-04-07.
INFO:__main__:Creating OU=OU_001 with path=R_001&OU_001 at level=1 on created_timestamp=2020-04-07.
INFO:__main__:Adding account='account 001' to org=OU_001 on created_timestamp=2020-04-07.
INFO:__main__:Adding account='account 002' to org=OU_001 on created_timestamp=2020-04-07.
INFO:__main__:Creating OU=OU_002 with path=R_001&OU_002 at level=1 on created_timestamp=2020-04-07.
INFO:__main__:Adding account='account 003' to org=OU_002 on created_timestamp=2020-04-07.
INFO:__main__:Creating OU=OU_003 with path=R_001&OU_002&OU_003 at level=2 on created_timestamp=2020-04-07.
INFO:__main__:Creating OU=OU_004 with path=R_001&OU_004 at level=1 on created_timestamp=2020-04-07.
INFO:__main__:Adding account='account 004' to org=OU_004 on created_timestamp=2020-04-07.
INFO:__main__:Creating OU=OU_005 with path=R_001&OU_002&OU_005 at level=2 on created_timestamp=2020-04-07.
INFO:__main__:Converted date_delta=-31 to 2020-06-05
INFO:__main__:Updating account=account 004 with delete_timestamp=2020-06-05
INFO:__main__:Updating org=OU_004 with delete_timestamp=2020-06-05
INFO:__main__:Converted date_delta=0 to 2020-07-06
INFO:__main__:Adding account='account 003' to org=OU_003 on created_timestamp=2020-07-06.
INFO:__main__:Creating OU=OU_005 with path=R_001&OU_001&OU_005 at level=2 on created_timestamp=2020-07-06.
```

Reply:
Thanks @myersCody 😄